packages/core/src/projects/index.ts

```diff
@@ -617,7 +617,7 @@ export const versionCheck = async (c: RnvContext) => {
         `versionCheck:rnvRunner:${c.runtime.rnvVersionRunner},rnvProject:${c.runtime.rnvVersionProject}`,
         chalk().grey
     );
-    if (c.runtime.rnvVersionRunner && c.runtime.rnvVersionProject) {
+    if (c.runtime.rnvVersionRunner && c.runtime.rnvVersionProject && c.runtime.rnvVersionRunner !== 'unknown') {
```

Comment:
@mihaiblaga89 this workaround looks weird. we're already checking `c.runtime.rnvVersionProject` which in this case is present. and it get's it from `c.files.project?.package?.devDependencies?.rnv` 

adding `c.runtime.rnvVersionRunner !== 'unknown'` seems like a ducktape of symptom rather than tackling actual culprit.

we need to understand why is `c.files.rnv.package` not populated at this point instead
first check should be `c.paths.rnv.dir` value and what's going on in `createRnvContext`

can you provide repro case hod did you arrived at this stage

Reply:
@mihaiblaga89 also add link to related GH issue this PR is meant to solve/fix

Reply:
@pavjacko that's correct, rnvVersionProject should have been checked for `unknown`. The problem was that the project I was using this in was not a renative project and had `rnv` as a dep, not devDep. Added dep possibility